app/serializers/serialize_exercise_instructions.rb

```diff
@@ -13,8 +13,7 @@ def call
 
   private
   def overview
-    # TODO: remove once we figured out how to handle practice exercises
-    return '' if exercise.practice_exercise?
+    return instructions if exercise.practice_exercise?
```

Comment:
This isn't correct I don't think, because practice_exercises might have introductions too, which need to be rendered here?

Reply:
Introductions are passed to the editor via a different proeprty